metadata-builders/src/main/java/com/linkedin/metadata/builders/search/DataProcessIndexBuilder.java

```diff
@@ -0,0 +1,31 @@
+package com.linkedin.metadata.builders.search;
+
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.metadata.search.DataProcessDocument;
+import com.linkedin.metadata.snapshot.DataProcessSnapshot;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+public class DataProcessIndexBuilder extends BaseIndexBuilder<DataProcessDocument> {
+
+    public DataProcessIndexBuilder() {
+        super(Collections.singletonList(DataProcessSnapshot.class), DataProcessDocument.class);
+    }
+
+    @Nullable
+    @Override
+    public List<DataProcessDocument> getDocumentsToUpdate(@Nonnull RecordTemplate snapshot) {
+        return null;
```

Comment:
I would suggest to throw exception here as well instead of silently returning null.